src/style.css

```diff
@@ -0,0 +1,30 @@
+
+* {
+  margin: 0;
+}
```

Comment:
don't reset your margins with * selector, it's a bad practice, you only need to reset body margin in this task